main-app/app/scripts/services/directives.js

```diff
@@ -1,20 +1,22 @@
-(function(){
-//Todo: Square Selection Directive
-angular.module('noughtsAndCrossesApp')
+/*(function(){*/
+
+angular.module('tombola.noughtsAndCrossesApp.directives.gameSquare')
     .directive('gameSquare', function  (){
      return {
          restrict:   'E',
          replace:    true,
          transclude: false,
          template:   function(scope, attr) {
-             return '<img src="" class="gameSquare {{ gameModel.gameState[' + attr.squarenumber + '] | playerNumberToGamePiece }}" ng-click="makeMove(' + attr.squarenumber + ')">';
+
+             return '<img src="" class="gameSquare {{ gameModel.gameState[' + attr.squarenumber + ']  | playerNumberToGamePiece }}" ng-click="makeMove(' + attr.squarenumber + ')">';
          },
      };
     });
-}());
-//Todo: Player Selection Directive
-(function(){
-    angular.module('noughtsAndCrossesApp')
+/*}());*/
+
+/*(function(){*/
+
+    angular.module('tombola.noughtsAndCrossesApp.directives.playerSelection')
```

Comment:
This should be just 'tombola.noughtsAndCrossesApp.directives' - the directive itself is playerSelection